playbooks/setup_jenkins_slave.yml

```diff
@@ -1,7 +1,49 @@
 ---
 - hosts: job_nodes
   user: root
+  vars:
+    rpc_release_file_name: "{{ lookup('env', 'WORKSPACE') ~ 'rpc-openstack/rpcd/etc/openstack_deploy/user_rpco_variables_defaults.yml' }}"
   tasks:
+
+    # We first check whether the file exists, as the job may
+    # not include a checkout of rpc-openstack
+    - name: Check if release file name exists
```

Comment:
Most if not all jobs will run this setup before they would checkout rpc-openstack since the CIT Jenkins slave generally only needs to pull rpc-gating. Additionally, the checkouts would be on the CIT slave instead of the the public cloud server that is currently being configured, so these tasks should be delegated to localhost instead of running on the job_nodes hosts.